AndieDrew/travel-tracker
# Feature/book trip

## Is this a fix or a feature?

- Feature

## What is the change?

- User can now check the price of a trip, book the trip, and see the trip added to their list of trips as pending.

## What does it fix?

- Some functionality was refactored for scope and to be more dynamic.

## Where should the reviewer start?

- index.js 

## How should this be tested?

- open application and fill out the form. Check to see if form is prevented from being submitted unless all fields are filled out. Also check to see that calendar cannot select dates in the past. Check that the check price feature works and than when clicking the book trip submit button that it displays at the bottom of the trips list.
